src/assets/css/material/_main.scss

```diff
@@ -315,9 +315,9 @@ mat-form-field.mat-form-field-should-float:not(.mat-focused):not(.ng-invalid):no
 
 .mat-select-panel {
   margin-left: 6px;
-  margin-top: 30px;
-  max-width: calc(100% + 20px);
-  min-width: calc(100% + 20px);
+  margin-top: 38px;
+  max-width: calc(100% + 20px) !important;
```

Comment:
Is max-width applied programmatically on this element to use important?

Reply:
Yes, through `style` attribute.